pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -206,6 +206,20 @@ type GracefulEvictionTask struct {
 	// +required
 	Producer string `json:"producer"`
 
+	// GracePeriodSeconds is the maximum waiting duration in seconds before the item
+	// should be deleted. If the application on the new cluster cannot reach a Healthy state,
+	// Karmada will delete the item after GracePeriodSeconds is reached.
+	// Value must be positive integer.
+	// It can not co-exist with SuppressDeletion.
+	// +optional
+	GracePeriodSeconds *int32 `json:"gracePeriodSeconds,omitempty"`
+
+	// SuppressDeletion represents the grace period will be persistent until
+	// the tools or human intervention stops it.
+	// It can not co-exist with GracePeriodSeconds.
+	// +optional
+	SuppressDeletion *bool `json:"suppressDeletion,omitempty"`
```

Comment:
Can we just set its type to bool?

Reply:
We should distinguish between `==nil` and `false` these two cases. if `SuppressDeletion` is nil, we will use `Graciously` mode to deal with tasks. If `SuppressDeletion` == false, it means user want to delete the task now.

Reply:
ok, I get it.